test/sandbox/app/components/helpers_api_component.html.slim

```diff
@@ -0,0 +1,2 @@
+.helper__message
```

Comment:
Our convention is to write test templates in ERB unless we're testing something template engine-specific.

Reply:
I have rewritten it in erb